arm-sql/2014-04-01/swagger/auditingPolicies.json

```diff
@@ -0,0 +1,410 @@
+{
+    "swagger": "2.0",
+    "info": {
+        "title": "Azure Sql Database API spec",
+        "description": "The Azure SQL Database management API provides a RESTful set of web services that interact with Azure SQL Database services to manage your databases. The API enables users to create, retrieve, update, and delete databases.",
+        "version": "2014-04-01"
+    },
+    "host": "management.azure.com",
+    "schemes": [
+        "https"
+    ],
+    "consumes": [
+        "application/json"
+    ],
+    "produces": [
+        "application/json"
+    ],
+    "paths": {
+        "/subscriptions/{subscriptionId}/resourceGroups/{resourceGroupName}/providers/Microsoft.Sql/servers/{serverName}/databases/{databaseName}/auditingPolicies/Default": {
+            "put": {
+                "tags": [
+                    "AuditingPolicy"
+                ],
+                "operationId": "AuditingPolicy_CreateOrUpdateDatebasePolicy",
+                "description": "Creates or updates an Azure SQL Database auditing policy.",
+                "parameters": [
+                    {
+                        "$ref": "#/parameters/ApiVersionParameter"
+                    },
+                    {
+                        "$ref": "#/parameters/SubscriptionIdParameter"
+                    },
+                    {
+                        "$ref": "#/parameters/ResourceGroupParameter"
```

Comment:
@amarzavery, @tbombach, @johanste - Is it recommended to match parameter name with the corresponding name in the url? In this case, url has `ResourceGroupName` while this one has `ResourceGroupParameter`. Should it rather be `ResourceGroupNameParameter`?